api.md

```diff
@@ -0,0 +1,188 @@
+API Documentation
+===
+* [**Builder**](#builder)
+* [**Expression Strings**](#module-tree-expressions)
+
+Builder
+---
+* [**new Builder()**](#new-builderconfig)
+* [**config()**](#builderconfigconfig-saveforreset-ignorebaseurl)
+* [**loadConfig()**](#builderloadconfigconfigfile-saveforreset-ignorebaseurl)
+* [**loadConfigSync()**](#builderloadconfigsyncconfigfile-saveforreset-ignorebaseurl)
+* [**reset()**](#builderconfigconfig-saveforreset-ignorebaseurl)
+* [**bundle()**](#builderconfigconfig-saveforreset-ignorebaseurl)
+* [**buildStatic()**](#builderconfigconfig-saveforreset-ignorebaseurl)
+* [**trace()**](#builderconfigconfig-saveforreset-ignorebaseurl)
+* [**addTrees()**](#builderaddtreesfirsttree-secondtree)
+* [**subtractTrees()**](#buildersubtracttreesfirsttree-secondtree)
+* [**intersectTrees()**](#builderintersecttreesfirsttree-secondtree)
+
+### new Builder([config])
+### new Builder([baseURL, [configFile]])
+`config`: An object conforming to the [config api] (https:/github.com/systemjs/systemjs/blob/master/docs/config-api.md)  
+`baseURL`: Sets the root for the loader  
+`configFile`: A systemjs config file conforming to the [systemjs config api] (https:/github.com/systemjs/systemjs/blob/master/docs/config-api.md)  
+#### Example
+```javascript
+new Builder({
+  map: {
+    'a': 'b.js'
+  }
+});
+
+new Builder('scripts', 'config.js');
+```
+
+### builder.config(config[, saveForReset[, ignoreBaseURL]])
```

Comment:
Makes sense, I included it since I had noticed a comment about it being part of the api, https://github.com/systemjs/builder/blob/master/lib/builder.js#L313